### ISSUE TITLE
A request for Regex - minor change

### ISSUE THREAD

**Issue by dummifiedme** [Pseudonium/Obsidian_to_Anki#163]
I am using regex based on `#flashcard`. I use several such regexes based on the type of cards, and there is this small issue.
The regex currently searches for a `\n` character at the end. But, sinch #tags are inherent tagging system of Obsidian it suggests and autocompletes the tag (which is great), but it adds a space as well. 
Of course I can try to take it out, but by chance if I miss it, the card is lost without errors! And also, in Obsidian after autocompletion if I hit backspace and press enter, it again adds a space and stays there (lol). Basically we have to a manual movement to go into next line.

Can this be changed via regex to basically an OR kinda thing between a `space+\n` and just `\n`?

Thanks :)

**Comment by Pseudonium** [comment]
Yeah this is very easy, took like 10 seconds to change lol. Check the wiki for the updated regexp

**Comment by dummifiedme** [comment]
Thanks a lot. 👍

**Comment by dummifiedme** [comment]
On the same topic, can there be another slight change..

When using #flashcard type regex, it messes up with the headers. I mean, the tags remain as a suffix. Which isn't neat when I link the headers in my index or any there else.

I mean, it then looks like
Title page > This is a sub header flashcard

The flashcard word there is useless and weird when read in sentences, and there would be lot of them when I put links in my index or anywhere.


So, I don't exactly know which way to proceed.

Either this...

---

## This is my question header.
#tag1 #flashcard
This is my answer and all the other contents.
This is second line of the content to be included in the flashcard.

This is content that isn't part of card. 

---

OR

---

## This is my question header.
This is my answer and all the other contents.
This is second line of the content to be included in the flashcard. #tag1 #flashcard 

This is content that isn't part of card. 

---

Second one seems more suitable when it comes to me. But which ever is possible, can you please create a regex?

Basically, if there is a header (any level), then #flashcard (or any other name) after the content, it breaks the whole thing into two groups. The header line as question, and the other as answer.
It might not be the same thing or improvement over the neuracache way we have already, but another card type maybe? 😅

The card end can be indicated by two newline if that's required.
